docs/tutorials/scrape_websites_using_the_sitemap.md

```diff
@@ -0,0 +1,123 @@
+
+## [](#the-sitemap) The sitemap
+
+The sitemap is usually located at the path `/sitemap.xml`. It is always worth trying that URL, as it is rarely linked anywhere on the site. It usually contains a list of all pages in [XML format](https://www.w3.org/standards/xml/core).
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+        <loc>http://www.brewbound.com/advertise</loc>
+        <lastmod>2015-03-19</lastmod>
+        <changefreq>daily</changefreq>
+    </url>
+    <url>
+    ...
+```
+
+The URLs of breweries are in the form
+
+```cURL
+http://www.brewbound.com/breweries/[BREWERY_NAME]
+```
+
+and the URLs of craft beers are in the form
+
+```cURL
+http://www.brewbound.com/breweries/[BREWERY_NAME]/[BEER_NAME]
+```
+
+They can be matched with the following regular expression.
+
+```cURL
+http:\/\/www\.brewbound\.com\/breweries\/[^\/]+\/[^\/<]+
+```
+
+Note the two parts of the regular expression `[^\/<]` containing `<`. This is because we want to exclude the `</loc>` tag, which closes each URL.
+
+## [](#using-the-sitemap-in-apify-sdk) Using the sitemap in Apify SDK
+
+Our [web scraping and automation library](https://sdk.apify.com) is well-suited for scraping with sitemaps.
+
+First, let's import the beer URLs from the sitemap to [RequestList](https://sdk.apify.com/docs/api/request-list#docsNav) using our regular expression to match only the (craft!) beer URLs and not pages of breweries, contact page, etc.
+
+```javascript
+const requestList = await new Apify.RequestList({
+    sources: [{
+        requestsFromUrl: 'https://www.brewbound.com/sitemap.xml',  
+        regex: /http:\/\/www.brewbound.com\/breweries\/.*/,
+    }],
+});
+
+await requestList.initialize();
+```
+
+Now, let's use [PuppeteerCrawler](https://sdk.apify.com/docs/api/puppeteer-crawler#docsNav) to scrape the created [RequestList](https://sdk.apify.com/docs/api/request-list#docsNav) with [Puppeteer](https://pptr.dev) and push it to the final dataset.
+
+```javascript
+const crawler = new Apify.PuppeteerCrawler({
+    requestList,
+    handlePageFunction: async ({ page, request }) => {
+        const data = await page.evaluate(() => {
+            const title = document.getElementsByTagName('h1')[1].innerText;
+            const [brewery, beer] = title.split(':');
+            const description = document.getElementsByClassName('productreviews')[0].innerText;
+
+            return { brewery, beer, description };
+        });
+
+        await Apify.pushData(data);
+    },
+});
+
+await crawler.run();
+```
+
+## [](#full-code-example) Full code example
+
+If we create a new actor using the code below on the Apify [platform](https://my.apify.com/actors/9WLZBWCwS3a5izqit#/source), it returns a nicely formatted spreadsheet containing a list of breweries with their beers and descriptions.
+
+```javascript
```

Comment:
I will add note that user needs to use apify/actor-node-chrome base image, otherwise run fails.